src/04-accordion/style.scss

```diff
@@ -10,9 +10,12 @@
 // https://design-system.service.gov.uk/components/accordion/
 @import "node_modules/govuk-frontend/govuk/components/accordion/accordion";
 
+// https://design-system.service.gov.uk/components/details/
+@import "node_modules/govuk-frontend/govuk/components/details/details";
```

Comment:
Hey Adam. Did mean for this import to be here?

Reply:
Actually just read your comment :o)

Reply:
yes, it's used to bring in the arrows which is part of Toni's designs but unfortunately not part of the gds accordion design.